src/here-xyz.ts

```diff
@@ -878,7 +760,7 @@ function replaceOpearators(expr: string) {
 
 program
     .command("show <id>")
-    .description("shows the content of the given [id]")
+    .description("shows the content of the given space or project [id]")
```

Comment:
please remove project

Reply:
Have removed reference of projects (show / delete) from here-xyz.ts